.drone.star

```diff
@@ -631,81 +1021,137 @@ def phptests(testType):
 					command = 'make test-php-integration'
 
 			for db in params['databases']:
-				keyString = '-' + category if params['includeKeyInMatrixName'] else ''
-				name = '%s%s-php%s-%s' % (testType, keyString, phpVersion, db.replace(":", ""))
-				maxLength = 50
-				nameLength = len(name)
-				if nameLength > maxLength:
-					print("Error: generated phpunit stage name of length", nameLength, "is not supported. The maximum length is " + str(maxLength) + ".", name)
-					errorFound = True
-
-				result = {
-					'kind': 'pipeline',
-					'type': 'docker',
-					'name': name,
-					'workspace' : {
-						'base': '/var/www/owncloud',
-						'path': 'server/apps/%s' % config['app']
-					},
-					'steps':
-						installCore('daily-master-qa', db, False) +
-						installApp(phpVersion) +
-						installExtraApps(phpVersion, params['extraApps']) +
-						setupServerAndApp(phpVersion, params['logLevel']) +
-						setupCeph(params['cephS3']) +
-						setupScality(params['scalityS3']) +
-						params['extraSetup'] +
-					[
-						{
-							'name': '%s-tests' % testType,
+				for externalType in params['externalTypes']:
+					keyString = '-' + category if params['includeKeyInMatrixName'] else ''
+					filesExternalType = externalType if externalType != 'none' else ''
+					name = '%s%s-php%s-%s' % (testType, keyString, phpVersion, db.replace(":", ""))
+					maxLength = 50
+					nameLength = len(name)
+					if nameLength > maxLength:
+						print("Error: generated phpunit stage name of length", nameLength, "is not supported. The maximum length is " + str(maxLength) + ".", name)
+						errorFound = True
+
+					if (filesExternalType == ''):
+						# for the regular unit test runs, the clover coverage results are in a file named like:
+						# autotest-clover-sqlite.xml
+						coverageFileNameStart = 'autotest'
+						extraCoverageRenameCommand = []
+						extraCoverage = False
+					else:
+						# for the files-external unit test runs, the clover coverage results are in 2 files named like:
+						# autotest-external-clover-sqlite.xml
+						# autotest-external-clover-sqlite-samba.xml
+						coverageFileNameStart = 'autotest-external'
+						extraCoverageRenameCommand = [
+							'mv tests/output/coverage/%s-clover-%s-%s.xml tests/output/coverage/clover-%s-%s.xml' % (coverageFileNameStart, getDbName(db), externalType, name, externalType)
+						]
+						extraCoverage = True
+
+					result = {
+						'kind': 'pipeline',
+                        'type': 'docker',
```

Comment:
The indent changes here???

Reply:
Git is showing it as change even though no changes has been made in the indentation.